engine/execution/messages.go

```diff
@@ -17,18 +17,17 @@ type ComputationOrder struct {
 }
 
 type ComputationResult struct {
-	ExecutableBlock    *entity.ExecutableBlock
-	StateSnapshots     []*delta.SpockSnapshot
-	StateCommitments   []flow.StateCommitment
-	Proofs             [][]byte
-	Events             []flow.EventsList
-	EventsHashes       []flow.Identifier
-	ServiceEvents      flow.EventsList
-	TransactionResults []flow.TransactionResult
-	ComputationUsed    uint64
-	StateReads         uint64
```

Comment:
this was a meaning less metric, we care about state reads per collection level, so removed this and added other metrics to capture better signals.